src/legacy/core_plugins/kbn_doc_views/public/views/table/table.tsx

```diff
@@ -47,7 +48,7 @@ export function DocViewTable({
         {Object.keys(flattened)
           .sort()
           .map(field => {
-            const valueRaw = flattened[field];
+            const valueRaw = get(hit._source, field, flattened[field]);
```

Comment:
Get the value from `_source` if available else fallback to `flattened[field]` which would be mostly the case for meta fields such as `_index`, etc.